benchmarking/frame-weight-template.hbs

```diff
@@ -35,24 +20,9 @@
 use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
 use sp_std::marker::PhantomData;
 
-/// Weight functions needed for {{pallet}}.
-pub trait WeightInfo {
-	{{#each benchmarks as |benchmark|}}
-	fn {{benchmark.name~}}
-	(
-		{{~#each benchmark.components as |c| ~}}
-		{{c.name}}: u32, {{/each~}}
-	) -> Weight;
-	{{/each}}
-}
-
```

Comment:
Why removing the generation of  the `WeightInfo` trait?

Reply:
The trait should always be defined in the pallet and implemented in `runtime/common/src/weights/*`.

This is already the case, the template was just outdated: https://github.com/moonbeam-foundation/moonbeam/blob/06031d12c369aed299e4c1d0e177b9314e9dd9fe/runtime/common/src/weights/cumulus_pallet_xcmp_queue.rs#L47